Jenkinsfile

```diff
@@ -0,0 +1,137 @@
+@Library('common-shared') _
+
+pipeline {
+  agent {
+    kubernetes {
+      label 'kubedeploy-agent'
+      yaml '''
+      apiVersion: v1
+      kind: Pod
+      spec:
+        containers:
+        - name: kubectl
+          image: eclipsefdn/kubectl:1.9-alpine
+          command:
+          - cat
+          tty: true
+      '''
+    }
+  }
+
+  environment {
+    APP_NAME = 'geoip-rest-api'
+    NAMESPACE = 'foundation-internal-webdev-apps'
+    IMAGE_NAME = 'eclipsefdn/geoip-rest-api'
+    CONTAINER_NAME = 'geoip-rest-api'
```

Comment:
Should this say quarkus instead of geoip-rest-api?

Reply:
the CONTAINER_NAME is the name of the container in the pod. In the productino.yaml file, you specified "app" as the container name, so it should be "app" here as well.